stubs/pyinstaller/PyInstaller/building/build_main.pyi

```diff
@@ -0,0 +1,28 @@
+# Referenced in: https://pyinstaller.org/en/stable/hooks.html?highlight=get_hook_config#PyInstaller.utils.hooks.get_hook_config
+# Not to be imported during runtime, but is the type reference for hooks and analysis configuration
+
+from _typeshed import StrOrBytesPath
+from collections.abc import Iterable
+
+from PyInstaller.building.datastruct import Target
+
+class Analysis(Target):
```

Comment:
Given the reason I haven't completed this stub (see comment at the top of the file). Should I add a `# incomplete` comment here?
I have the same question for similar reasons in `depend/analysis.pyi`.